std/utils.asm

```diff
@@ -25,6 +25,26 @@ let unchanged_until: expr, expr -> constr = |c, latch| (c' - c) * (1 - latch) =
 /// Evaluates to a constraint that forces `c` to be either 0 or 1.
 let force_bool: expr -> constr = |c| c * (1 - c) = 0;
 
+/// Creates a new witness column that is constrained to the values 0 and 1.
+let new_bool: -> expr = constr || {
```

Comment:
```suggestion
let mk_bool: -> expr = constr || {
```

Reply:
probably just biased because SMT solvers do it like that, but I think this notation is also common in FP?